addon/globalPlugins/remoteClient/session.py

```diff
@@ -14,8 +13,21 @@
 from . import connection_info
 import hashlib
 
+
+if not (

```

Comment:
Can't this condition be simplified to just `hasattr(speech.commands, "_CancellableSpeechCommand")`?

Reply:
For now, yes. The intention here was to be more explicit behind the API change so if the minimum version is ever bumped up, these can be removed. Happy to change it either way.